docs/http_proxy.md

```diff
@@ -0,0 +1,40 @@
+
+HTTP Forward Proxy Support
+==========================
+
+It is possible to launch a kubernetes cluster from behind an http forward proxy ("corporate proxy").  To do so, you will need to configure the `egressProxy` for the cluster.
+
+It is assumed the proxy is already existing.  If you want a private topology on AWS, for example, with an proxy instead of a NAT instance, you'll need to create the proxy yourself.  See [Running in a shared VPC](run_in_existing_vpc.md).
+
+This configuration only manages proxy configurations for Kops and the Kubernetes cluster.  We can not handle proxy configuration for application containers and pods.
+
+## Configuration
+
+Add `spec.egressProxy` port and url as follows
+
+``` yaml
+spec:
+  egressProxy:
```

Comment:
We do have `egress` which is where we configure a natGateway.  Currently that is defined on the subnet level, because of the limitations of NAT subnets.  We should consider whether this is the same thing, and whether we want to expand egress to cover this.  My knee-jerk is yes...

(In practice, we only have to preserve serialized API compatibility, so this means renaming the field.  We can harmonize the two fields later.)